backend/routes/api.php

```diff
@@ -24,7 +25,7 @@
         return UserResource::make($request->user());
     });
     Route::apiResource('users', UserController::class)->only(['index']);
-    Route::get('/users/{user_id}/accounts', [UserController::class, 'myAccounts']);
+    Route::resource('users.accounts',UserController::class)->shallow();
```

Comment:
use apiResource, add only() chain

this is exposing unhandled routes
![image](https://github.com/framgia/sph-flutter/assets/95029803/73293200-d148-4bc0-b103-90970f02a31e)